### PR TITLE
docs: add @pondy007 as a contributor

### PR DESCRIPTION
**Pull request by Pondy007**
## Description

This PR adds _Pondy007_ as a contributor.

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [x] 📝 Documentation Update

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes
- [ ] ❌ Not yet

### How did you add yourself as a contributor?

- [ ] 🤖 With CLI
- [x] ⌨️ Manually

#### If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?

- [x] ✅ Yes
- [ ] ❌ No

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot

image

## [optional] What GIF best describes this PR or how it makes you feel?
image

### CODE REVIEW

**Comment by BekahHW** [comment]
Thanks for your Pull Request! We've merged in some new PRs since you submitted. Can you merge the latest changes and resolve conflicts? If you need some help with that, check out [these directions](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository).

**Review by BekahHW — DISMISSED** [review]
Can you merge in the latest branch and update your merge conflicts, please?

**Comment by Pondy007** [comment]
OK, I will try and keep up posted.

**Comment by Pondy007** [comment]
I have successfully merged with previous PR.
image

File: README.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## All Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-109-orange.svg?style=flat-square)](#contributors-)
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-110-orange.svg?style=flat-square)](#contributors-)
+
```
Comment (adiati98) [inline README.md:41]: Please remove the space before anad after the badge as suggestion below:

```suggestion
[![All Contributors](https://img.shields.io/badge/all_contributors-110-orange.svg?style=flat-square)](#contributors-)
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
This looks good, @Pondy007!
Congrats for successfully merging the conflicts! 🎉

There's only one more thing to fix, and this is good to go. 👍

File: README.md (hunk 1 above)
Comment (Pondy007) [reply README.md:41]: Empty spaces removed as requested

**Review by adiati98 — APPROVED** [review]
Well done, @Pondy007! 🙌
Thank you for the changes! 😄

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Comment by Pondy007** [comment]
Thank you @adiati98 for your approval.
I have also joined the community in discord (but new to that platform).

**Review by CBID2 — APPROVED** [review]
Merge time! :)

**Pull request merged**